packages/botonic-react/src/components/message.jsx

```diff
@@ -131,7 +135,7 @@ export const Message = props => {
         data: decomposedChildren ? decomposedChildren : textChildren,
         timestamp: props.timestamp || getFormattedTimestamp,
         markdown,
-        from,
+        sentBy: props.sentBy,
```

Comment:
Nit: You could use `sentBy` directly, am I right?
```suggestion
        sentBy,
```

Reply:
Applied